src/item.cpp

```diff
@@ -2432,11 +2432,17 @@ int item::volume( bool integral ) const
         }
     }
 
-// tool mods also add volume
-    if (has_flag("ATOMIC_AMMO")) {
+    // Battery mods also add volume
+    if( has_flag("ATOMIC_AMMO") ) {
         ret += 1;
     }
 
+    if( has_flag("DOUBLE_AMMO") ) {
+        // Batteries have volume 1 per 100 charges
+        // TODO: De-hardcode this
```

Comment:
`ammo_data() ? ammo_data()->id : default_ammo( ammo_type() )` or something to that effect